roles/github/defaults/main.yml

```diff
@@ -36,25 +40,34 @@ github_tempest_test_suites: |
   - tempest-full
 
 github_kayobe_limit_input: |
-  kayobeLimit:
+  kayobe_limit:
     description: |
       The ansible limit to use when running kayobe playbooks.
 
 github_kayobe_tags_input: |
-  kayobeTags:
+  kayobe_tags:
     description: |
       The ansible tags to use when running kayobe playbooks.
 
 github_kolla_limit_input: |
-  kollaLimit:
+  kolla_limit:
     description: |
       The ansible limit to use for kolla-ansible playbooks.
 
 github_kolla_tags_input: |
-  kollaTags:
+  kolla_tags:
     description: |
       The ansible tags to use when running kolla-ansible playbooks.
 
+github_kayobe_environment_input: |
+  kayobe_environment:
+    description: |
+      Select the environment the kayobe workflow shall target.
+    type: choice
+    required: true
+    default: 'production'
+    options:
```

Comment:
nit: would it work to template the environment list here?

Reply:
Cannot template within the string however could manage it by breaking the string apart. Will update with a solution.

Reply:
I'm fairly sure you can template within the string

Reply:
Noticed an issue between different ansible versions (ansible-core ~2.14) where it would fail to template variables inside of string. However works fine in ansible-core 2.15 and in the test-collection workflow.